src/analysis/normed_space/banach.lean

```diff
@@ -103,7 +104,17 @@ begin
             end
           ... = (ε / 2)⁻¹ * ∥c∥ * 2 * ↑n * ∥y∥ : by ring,
         exact ⟨d⁻¹ • x, J, 𝕜⟩ } } },
-  rcases this with ⟨C, C0, hC⟩,
+end
```

Comment:
remove braces and unindent from line 48 to line 106

Reply:
Done.